### PR TITLE
Added evidence, more codes, from notes taken during pass through data…

### PR DESCRIPTION
**Pull request by kisaacs**
… interface.

### CODE REVIEW

File: codes.md [hunk 1]
```diff
@@ -76,11 +94,12 @@ Codes
 ## FOREIGN_TERMINOLOGY. Even very technical people find some data abstraction concepts, language foreign
   - One Supercomputing survey participant was confused by "item" vs "attribute" terminology. In their response, "tens of items" was selected (initially thinking about table columns), but then selected millions after discussing with A-1, A-3
   - Other generalizations: "A spreadsheet," even though "Tabular" data was an option and we gave them the opportunity to rename it (although they may not have known)!
+  - “I had a hard time, actually, thinking of the dataset as tabular in a way that I had not previously thought of it as tabular (which I believe was the intent of this survey). The questions did not seem to specifically account for this possibility as distinct from considering it as tabular for the first time.” (from Alternative Dataset Debrief) -- A-3: This may be an indicator that the language use is founded in tables, already doesn't fit some datasets
```
Comment (alex-r-bigelow) [inline codes.md:97]: This one is really interesting—this definitely fits as evidence for this code, but the "first time" angle seems like there might be another one, especially in light of the conversation we just had on Slack. I may reuse this elsewhere...

File: codes.md [hunk 2]
```diff
@@ -125,19 +145,115 @@ Codes
...
+    - “There are links to media in the dataset, but the media themselves are not directly stored in the dataset.” (from Initial Media Data Details)
+  -  “the data in this way is more complicated to describe, but in the real world is more offen.” (from Alternative Dataset Reflections)
+  -  “Oftentimes, for certain types of time-series meteorological data is collected from one sensor that logs three values at one time stamp: min, max, avg. This does provoke an idea that data at certain time steps are related and can be related in a "tree like structure" but it also causes problems in visualizing the data because metadata is encoded along with data. This may muddle the purity of visualizing the data by itself.” (from Alternative Dataset Reflections) -- note the use of the word "muddle"
+  - Counts of how many people listed their data under multiple forms
+    - Media was the largest alternative category --- was it because it was the
+      only choice remaining for some participants?
+
+## LATENT_WRANGLING_OR_ABSTRACTION. This may be related to "HARD_TO_ISOLATE". Even before the alternative abstractions, people were discussing how they could see data in other forms.
```
Comment (alex-r-bigelow) [inline codes.md:196]: This is probably a better way to phrase the AUTOMATIC_DERIVATION_INFERENCE code; any objections if I put its evidence under this header?

**Review by alex-r-bigelow — APPROVED** [review]

File: codes.md (hunk 2 above)
Comment (alex-r-bigelow) [reply codes.md:196]: (I can do that in a later PR)

Comment (kisaacs) [reply codes.md:196]: Yes, definitely move things around/re-word as they make sense.

**Pull request merged**